tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/ValidationFunctions.cs

```diff
@@ -372,11 +372,11 @@ private GenericResponseVM MatterNameValidation(Matter matter)
                 }
                 else
                 {
-                    var matterDescription = Regex.Match(matter.Description, matterSettings.SpecialCharacterExpressionMatterDescription, RegexOptions.IgnoreCase);
-                    if (matter.Description.Length > matterSettings.MatterDescriptionLength  || !matterDescription.Success)
-                    {
-                        return GenericResponse(errorSettings.IncorrectInputMatterDescriptionCode, errorSettings.IncorrectInputMatterDescriptionMessage);
-                    }
+                    //var matterDescription = Regex.Match(matter.Description, matterSettings.SpecialCharacterExpressionMatterDescription, RegexOptions.IgnoreCase);
```

Comment:
Removed the regular expression validation for Matter Description and now the matter description will allow all the special characters